src/main/java/com/bravo/user/controller/AddressController.java

```diff
@@ -0,0 +1,40 @@
+package com.bravo.user.controller;
+
+import java.util.List;
+
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import com.bravo.user.annotation.SwaggerController;
+import com.bravo.user.model.dto.AddressDto;
+import com.bravo.user.service.AddressService;
+import com.bravo.user.validator.UserValidator;
+
+@RequestMapping(value = "/address")
+@SwaggerController
+public class AddressController {
+
+	private final AddressService addressService;
+	private final UserValidator userValidator;
+
+	public AddressController(AddressService addressService, UserValidator userValidator) {
+		this.addressService = addressService;
+		this.userValidator = userValidator;
+	}
+
+	@GetMapping(value = "/retrieve/{userId}")
+	@ResponseBody
+	public List<AddressDto> retrieve(final @PathVariable String userId,
+			final @RequestParam(required = false) Integer page,
+			final @RequestParam(required = false) Integer size,
```

Comment:
You're accepting the `page` and `size` parameters, but not using them.

Reply:
good catch, I initially was going to implement pagination until I re-read the requirements to simply return all addresses for the userId. I will clean them up, as well as the unused httpResponse parameter